server/src/study-room/study-room.controller.ts

```diff
@@ -1,21 +1,40 @@
-import { Body, Controller, Get, Post } from '@nestjs/common';
+import { Body, Controller, Get, Inject, ParseIntPipe, Post, Query } from '@nestjs/common';
 import { StudyRoomsService } from './study-room.service';
 import { StudyRoom } from './entity/study-room.entity';
+import { WINSTON_MODULE_PROVIDER } from 'nest-winston';
+import { Logger } from 'winston';
 
 @Controller('study-room')
 export class StudyRoomController {
-  constructor(private readonly studyRoomService: StudyRoomsService) {}
+  constructor(
+    @Inject(WINSTON_MODULE_PROVIDER)
+    private readonly logger: Logger,
+    private readonly studyRoomService: StudyRoomsService,
+  ) {}
 
   @Post('/create')
   async creatRoom(
-    @Body('roomId') roomId: string,
+    @Body('roomName') roomName: string,
     @Body('clientId') clientId: string,
   ): Promise<StudyRoom> {
-    return await this.studyRoomService.createRoom(roomId, clientId);
+    return await this.studyRoomService.createRoom(roomName, clientId);
```

Comment:
async await 안걸고 그냥 return 해도 똑같이 promise 리턴할 수 있을거 같아요
```suggestion
    return this.studyRoomService.createRoom(roomName, clientId);
```